### PR TITLE
Update June meetup talks

### PR DESCRIPTION
**Pull request by akshay196**
Add talks from June Python Pune meetups.

### CODE REVIEW

File: 2019/June/README.md [hunk 1]
```diff
@@ -1 +1,34 @@
...
+## Logging in Python
+  * **Speaker**: Akshay Gaikwad
+  * **Description**: In simple term, logging is a way to understand
+    what is going on inside an application. It can help programmer to
+    understand flow of a program. With logging, it is easy to
+    constantly look at the flow that an application going
+    through. Python has simple and powerful logging module as standard
+    library from 2.3, which described in PEP-282.
+
+  * [Slides](https://gitlab.com/akshay196/slides/blob/master/logging/presentation.org)
```
Comment (bhavin192) [inline 2019/June/README.md]: How about adding source here as well? That will give more clarity.

File: 2019/June/README.md [hunk 2]
```diff
@@ -1 +1,34 @@
...
+  * **Speaker**: Bhavin Gandhi
+  * **Description**: We spend most of our time writing things, be it
+    code, documentation, blog post, emails etc. The tool that one uses to
+    write things should be something which is highly configurable and
+    focuses more on productivity. This talk will cover reasons why one
+    should use GNU Emacs, it will cover various things Emacs can do out of
+    the box. How it's possible to extend it for various programming
+    languages and editing needs.
+
+  * [Source](https://github.com/pythonpune/meetup-talks/issues/11)
```
Comment (bhavin192) [inline 2019/June/README.md]: Can you please make these changes?

```suggestion
  * [Slides](https://bhavin192.gitlab.io/talks/2019/june-python-pune-meetup/gnu-emacs-talk.html)
  * [Slides with notes](https://gitlab.com/bhavin192/talks/raw/master/2019/june-python-pune-meetup/gnu-emacs-talk.org)
  * [Source](https://github.com/pythonpune/meetup-talks/issues/11)
```

File: 2019/June/README.md (hunk 1 above)
Comment (akshay196) [reply 2019/June/README.md]: Done

File: 2019/June/README.md (hunk 2 above)
Comment (akshay196) [reply 2019/June/README.md]: Done

File: 2019/June/README.md (hunk 1 above)
Comment (akshay196) [reply 2019/June/README.md]: Done

File: 2019/June/README.md (hunk 2 above)
Comment (akshay196) [reply 2019/June/README.md]: Done

**Comment by bhavin192** [comment]
This fixed #10 #11

**Pull request merged**